plugin/trino-hive/src/main/java/io/trino/plugin/hive/TrinoViewHiveMetastore.java

```diff
@@ -87,7 +87,7 @@ public void createView(ConnectorSession session, SchemaTableName schemaViewName,
 
         Optional<io.trino.plugin.hive.metastore.Table> existing = metastore.getTable(schemaViewName.getSchemaName(), schemaViewName.getTableName());
         if (existing.isPresent()) {
-            if (!replace || !isPrestoView(existing.get())) {
```

Comment:
In theory, the old code could replace an MV with a View.

Reply:
Could we add a test to verify it isn't allowed now ?

Reply:
it would be very hard to write such a test. from linked PR desc

> This would have only been possible during a rare race condition since the engine also checks that an existing entity is not a MV before calling on the connector to create or replace the view.